src/arduino/vampires_trigger/vampires_trigger.ino

```diff
@@ -138,17 +139,19 @@ void trigger_loop_flc() {
     // as a means of signalling the failure.
     if (!camera_one_ready || !camera_one_ready) disable_loop();
     
-    // Sleep 60 micros to create +/- 30 timing jitter.
-    for (start_time = micros(); micros() - start_time < 60;) continue;
+    // Sleep 30 micros before FLC active to create +/- 30 timing jitter.
+    for (start_time = micros(); micros() - start_time < jitter_half_width;) continue;
 
     // start second half- reset variables
     camera_one_ready = camera_two_ready = false;
     // Second exposure FLC is active
     digitalWrite(FLC_TRIGGER_PIN, HIGH);
-    for (start_time = micros(); micros() - start_time < flc_offset + trig_delay;) continue;
+    // Sleep 30 micros after FLC active to create +/- 30 timing jitter
+    // in addition to the FLC offset and extra delay
+    for (start_time = micros(); micros() - start_time < flc_offset + trig_delay + jitter_half_width;) continue;
     // Send camera pulse
     digitalWrite(CAMERA_TRIGGER_PIN, HIGH);
-    for (;micros() - start_time < pulse_width + flc_offset + trig_delay;) continue;
+    for (;micros() - start_time < pulse_width + flc_offset + trig_delay + jitter_half_width;) continue;
```

Comment:
Why is there a jitter half-width here? There's already two of them above, and this is during exposure :o ??

Reply:
Just ugly math

start_time is counting total time since FLC trigger high to account for 1s maximum. This means I'm accumulating the total offset since FLC high. Which includes 
- one half width
- the delay
- the flc offset
Camera trigger goes high now
- The pulse width 
At the time the camera trigger goes low

Reply:
Oh K my bad I thought start_time was being accounted for each while loop. Just ugly reading :)